src/components/busEstimate/busEstimate.test.tsx

```diff
@@ -1,36 +1,40 @@
 import React from 'react';
 import BusEstimate from './busEstimate';
-import { mount } from 'enzyme';
-import Enzyme from 'enzyme';
 import StubBusTrackerAPI from '../../Services/stubBusTrackerAPI';
-import Adapter from 'enzyme-adapter-react-16';
-import toJSON from 'enzyme-to-json';
-
-function flushPromises() {
-    return new Promise(resolve => setImmediate(resolve));
-}
-
-describe('<BusEstimate />', () => {
-    Enzyme.configure({ adapter: new Adapter() })
-    it('does a snapshot', () => {
-        const wrapper = mount(
-            <BusEstimate api={StubBusTrackerAPI} />
-        );
-
-        expect(toJSON(wrapper)).toMatchSnapshot();
-    });
-
-    it('should update text when clicking on button', () => {
-        const wrapper = mount(
-            <BusEstimate api={StubBusTrackerAPI} />
-        );
-
-        const button = wrapper.find('#estimateButton').at(0);
-        button.simulate('click');
-
-        return flushPromises().then(() => {
-            wrapper.update()
-            expect(toJSON(wrapper)).toMatchSnapshot();
-        });
-    });
+import { render, fireEvent, cleanup, waitForElement } from '@testing-library/react'
+
+
+afterEach(cleanup)
+it("Displays time estimate when route, direction, and stop are selected", async () => {
```

Comment:
Did I mention [prettier](https://github.com/prettier/prettier) already?  There's a VS Code plugin you can add which will auto-format all JS/TS on save.  It's pretty nice to get used to and use in teams, because it basically makes all stylistic choices for you; at this point I think much of the JS community uses it.